utils/scripts/ocm/ocm.py

```diff
@@ -412,6 +411,120 @@ def uninstall_rhods(self):
         """Uninstalls RHODS addon"""
         self.uninstall_addon(addon_name="managed-odh")
 
+    def install_addon_v2(self, addon_name="managed-odh",
```

Comment:
I could, but there are some differences which might cause issues on CI/CD, I wanted to talk to you about this. Let's sync up offline

Reply:
done it. I've added `exit_on_failure` parameter to `install_addon` and `uninstall_addon` functions to pilot the exit logic in case of errors:
- IF `exit_on_failure = True` THEN when any calls to ocm API fail, the execution is aborted, i.e., it runs `sys.exit(1)`. Required for Jenkins jobs to keep working as they do now. The param is defaulted to `True` to avoid changes on jenkins side. Robotframework texecution would be interrupted, no teardown would be run and no reports created.

- IF `exit_on_failure = False` THEN the execution is not aborted. Robotframework test will continue running (and likely fail). The report will be created

Reply:
setting this as "resolved" since we talked offline time ago. @aloganat pls re-open this conversation if needed